<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-PL6FGJN7" failures="1" tests="7" name="ChromeTest" time="33.001" errors="0" timestamp="2025-08-04T12:03:19 IST">
  <testcase classname="testCases.TC001_PurchaseFlow_Test" name="VerifyLoginFun" time="4.8"/>
  <testcase classname="testCases.TC001_PurchaseFlow_Test" name="VerifyAddToCartFunc" time="3.177"/>
  <testcase classname="testCases.TC001_PurchaseFlow_Test" name="Verify_CartPage_Func" time="3.433"/>
  <testcase classname="testCases.TC001_PurchaseFlow_Test" name="Verify_Checkout_Func" time="4.631"/>
  <testcase classname="testCases.TC001_PurchaseFlow_Test" name="Verify_Order_Success_Func" time="1.403"/>
  <testcase classname="testCases.TC001_PurchaseFlow_Test" name="Verify_Logout_Fun" time="0.422"/>
  <testcase classname="testCases.TC001_PurchaseFlow_Test" name="Verify_Invalid_Login_Fun" time="10.503">
    <failure type="java.lang.AssertionError" message="User able to login with invalid cred expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: User able to login with invalid cred expected [true] but found [false]
at testCases.TC001_PurchaseFlow_Test.Verify_Invalid_Login_Fun(TC001_PurchaseFlow_Test.java:124)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Verify_Invalid_Login_Fun -->
</testsuite> <!-- ChromeTest -->
